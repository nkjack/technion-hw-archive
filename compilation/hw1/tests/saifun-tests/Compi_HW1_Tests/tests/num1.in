1.1
0x1Q
0x
0x-F
0.cm
1.1.1%
15sec%
