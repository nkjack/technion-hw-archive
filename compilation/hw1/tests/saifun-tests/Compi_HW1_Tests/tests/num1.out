1 NUMBER 1
1 DOT .
1 NUMBER 1
2 NUMBER 0x1
2 NAME Q
3 UNIT 0x
4 UNIT 0x
4 NAME -F
5 NUMBER 0
5 DOT .
5 NAME cm
6 NUMBER 1
6 DOT .
6 UNIT 1.1%
7 UNIT 15sec
Error %
